openmc/stats/univariate.py

```diff
@@ -854,6 +855,17 @@ class Tabular(Univariate):
         Indicates how the density function is interpolated between tabulated
         points. Defaults to 'linear-linear'.
 
+    Notes
+    -----
+    The probabilities `p` are interpreted per unit of the corresponding independent 
+    variable `x`. This follows the definition of a probability density function (PDF)
+    in probability theory, where the PDF represents the relative likelihood of the 
+    random variable taking on a particular value per unit of the variable.
+
+    Example
+    -------
+    If `x` represents energy in eV, then `p` should represent probabilities per eV.
```

Comment:
Can you combine the "Example" section into the "Notes"? That is, just have a last sentence in the "Notes" section starting with "For example, if `x` represents ...". An "Example" section would really be intended to [illustrate usage of the class](https://numpydoc.readthedocs.io/en/latest/format.html#examples).